### DISCUSSION TITLE
How do I use recurring tasks?

### DISCUSSION THREAD

**Discussion by phenax** [nvim-neorg/neorg#455] [Q&A] [answered]
I'm trying out neorg and found recurring task pretty interesting but I can't seem to get it to work

```norg
   - [+] My task
   -- [ ] Sub-task 1
   -- [ ] Sub-task 2
```

Updating sub-task 1 to done, updates `[+]` to `[-]` and then it just behaves like a normal pending task. Am I missing something?

**Comment by max397574** [comment]
this i s no implemented yet

**Comment by mrossinek** [comment, marked as answer]
For the records: this discussion is being continued here: https://github.com/nvim-neorg/neorg/issues/457
